src/components/timeline/timeline.component.html

```diff
@@ -0,0 +1,3 @@
+<ul class="timeline hp-timeline-down-arrow">

```

Comment:
hp-timeline-down-arrow? Where did this come from?

Reply:
Can you remove it please? These styles have been retired since the HPE split.

Reply:
hp-timeline-down-arrow style removed.